src/Inquirer/Console/ConsoleManager.cs

```diff
@@ -24,6 +24,21 @@ public int[] Render(string[] content, string[] bottomContent)
             return new [] { content.Length, bottomContent.Length };
         }
 
+        public int[] Render(ConsoleMessage[] content, ConsoleMessage[] bottomContent)
+        {
+            content.ToList().ForEach(item =>
+            {
+                if (item.ConsoleColor.HasValue)
+                    console.ForegroundColor = item.ConsoleColor.Value;
+
+                console.WriteLine(item.Message);
+            });
+            Newline();
+            console.ResetColor();
```

Comment:
I think that this 'console.ResetColor()' should be inside the loop.
If we have 3 messages:

1. Console.Color = Cyan
1. Console.Color = (Not set)
1. Console.Color = Red

The second message will print Cyan, and I was expecting it to be the default Console color.

Could you please add some test to cover that? If need any help let me know